pymongo/asynchronous/mongo_client.py

```diff
@@ -2246,13 +2246,13 @@ async def bulk_write(
         :class:`~pymongo.operations.DeleteOne`, or
         :class:`~pymongo.operations.DeleteMany`).
 
-          >>> async for doc in await db.test.find({}):
+          >>> for doc in db.test.find({}):
```

Comment:
Sorry, we still need the `async for` here and below since `find` returns an asynchronous cursor.